plonk/src/multiprover/proof_system/constraint_system.rs

```diff
@@ -736,22 +887,46 @@ impl<C: CurveGroup> MpcCircuit<C> for MpcPlonkCircuit<C> {
         b: MpcVariable,
         c: MpcVariable,
     ) -> Result<(), MpcCircuitError> {
-        unimplemented!()
+        self.check_var_bound(a)?;
+        self.check_var_bound(b)?;
```

Comment:
missing a `self.check_var_bound(c)?;`:
```suggestion
        self.check_var_bound(a)?;
        self.check_var_bound(b)?;
        self.check_var_bound(c)?;
```